city_scrapers/spiders/il_elections.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime
+from urllib.parse import parse_qs, urlparse
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlElectionsSpider(CityScrapersSpider):
+    name = "il_elections"
+    agency = "Illinois State Board of Elections"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.elections.il.gov/AboutTheBoard/MeetingMinutesAll.aspx"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def __init__(self):
+        self.meeting_minutes = dict()
+        self.addresses = dict()
```

Comment:
This still doesn't need to be in the main `__init__` method and can just be passed as a regular dictionary argument to `_parse_location`